Large scale change: pre-commit checks vol2

kaidokert opened the pull request:
Runs `pre-commit --all` and fixes formatting in a number of low-churn, low risk files. This is according to current linter+formatter standards.

b/278726440

kaidokert commented:
First PR of this series in #1010

hlwarriner commented on .pre-commit-config.yaml:
```diff
@@ -122,8 +122,12 @@ repos:
             (?x)^(
                 .pre-commit-config.yaml$|
+                cobalt/base/unicode/|
+                cobalt/math/|
```
Why should this be excluded?

kaidokert replied:
Because this is not our code, it's all copyright Chrome and forked from there.

hlwarriner commented:
Can you please update the bug reference in the commit message? It looks like the wrong one.

kaidokert commented:
> Can you please update the bug reference in the commit message? It looks like the wrong one.

Thanks for catching this, updated the PR description - that's where the landed commit gets the bug ID now. I had dropped a zero ( b/278726440 is the correct one, or alternatively b/199280707 )
